MaxText/layers/embeddings.py

```diff
@@ -181,6 +181,7 @@ class LLaMARotaryEmbedding(RotaryEmbedding):
   # # LLaMA3.1 ROPE scaling, see the original pytorch implementation
   # https://github.com/meta-llama/llama-models/blob/301ca3a2b3b10e94ddcd1fdd2c57e52f812e1cac/models/llama3/reference_impl/model.py#L45C5-L45C18
   use_scale: bool = True
+  # timescale: Optional[Array] = None
```

Comment:
why is this added ?

Reply:
Was trying to resolve lint error "Instance attribute timescale defined outside __init__ "

Reply:
Was trying to resolve lint error "Instance attribute timescale defined outside `__init__`"